test/handlebars/standalone.test.js

```diff
@@ -0,0 +1,28 @@
+/* eslint no-console: 0 */
+/*global it, describe*/
```

Comment:
Is there a benefit to adding these? Other test files seem to be able to employ `it` and `describe` without this line.

Reply:
removed it, thanks for spotting it

Reply:
Still one instance in `test/handlebars/express.test.js`.